build.gradle

```diff
@@ -67,12 +67,12 @@ spotbugsTest {
 }
 
 ext {
-    opendistroVersion = '1.13.0'
+    opensearchVersion = '1.0.0'
```

Comment:
The `ext` block inside the `build` block defines an `opensearch_version` variable. I think we shouldn't need both. The version defined there is `1.0.0-alpha1`. Should we update that to be `beta1`?